pkg/strategy/support/strategy.go

```diff
@@ -386,8 +377,15 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 	s.orderStore = bbgo.NewOrderStore(s.Symbol)
 	s.orderStore.BindStream(session.UserDataStream)
 
-	if s.TrailingStopTarget.TrailingStopCallBackRatio != 0 {
-		s.trailingStopControl = NewTrailingStopControl(s.Symbol, s.Market, s.MarginOrderSideEffect, s.TrailingStopTarget.TrailingStopCallBackRatio, s.TrailingStopTarget.MinimumProfitPercentage)
+	if s.TrailingStopTarget.TrailingStopCallBackRatio.Float64() != 0 {
```

Comment:
float64 != 0 could be always true, because of the precison.

use s.TrailingStopTarget.TrailingStopCallBackRatio != 0